command/sub8_missions/readme.md

```diff
@@ -0,0 +1,112 @@
+Sub Mission Manager
+==================
+
+# Running the sub
+
+```shell
+    roslaunch sub8_launch sub8.launch
+```
+
+# Running the 3d mouse waypoint tool
+
+```shell
+    roslaunch sub8_launch waypoint_tool.launch
+```
+
+To get results that are useful, you'll want to set up rviz to visualize tf, /wrench, /posegoal, and /odom (Set odom and posegoal to use axes instead of arrow). I use /odom, keeping a bunch of arrows, for some indication that movement has actually happened.
+
+With the wire facing away from you, right-button will snap the posegoal to the sub's current pose, and left-button will make the sub go to the current posegoal
+
+tags for greppers: 3dmouse 3d-mouse 3dconnexion
+
+
+# Running a Mission
+
+```shell
```

Comment:
Make more clear, that these are example missions, and not the process to start a mission.